lib/kbuckets.ex

```diff
@@ -52,8 +52,10 @@ defmodule KBuckets do
   end
 
   def object(item = %Item{}) do
-    Model.KademliaSql.object(key(item))
-    |> Object.decode!()
+    case Model.KademliaSql.object(key(item)) do
+      nil -> nil
+      binary -> Object.decode!(binary)
+    end
```

Comment:
Same as below comment on `KademliaLight.network()` consistency -- if the db is not consistent this should crash and probably flush the memory table